src/Generator.h

```diff
@@ -2733,6 +2733,7 @@ class GeneratorBase : public NamesInterface, public GeneratorContext {
     Func get_first_output();
     Func get_output(const std::string &n);
     std::vector<Func> get_output_vector(const std::string &n);
+    std::vector<std::vector<Func>> get_output_vector();
```

Comment:
Took me a long time to figure out why this is a ```vector<vector<..>>``` These methods could use some brief descriptive comments.